pystac/extensions/sar.py

```diff
@@ -318,7 +318,7 @@ def ext(cls, obj: T, add_if_missing: bool = False) -> "SarExtension[T]":
             return cast(SarExtension[T], AssetSarExtension(obj))
         else:
             raise pystac.ExtensionTypeError(
-                f"SAR extension does not apply to type {type(obj)}"
+                f"SAR extension does not apply to type '{type(obj).__name__}'"
```

Comment:
Thanks for splitting this out into a separate PR. I like this change, and I think for consistency we should apply it to the `ext` methods of the other extensions as well.

Reply:
Done! I've also fixed the names of the extensions in some of them, which all had "File" in the error message.

Reply:
Thanks, good catch!